### DISCUSSION TITLE
Have no idea to debug quasar web app in webstorm

### DISCUSSION THREAD

**Discussion by luxi78** [quasarframework/quasar#10771] [General - Components / Directives / etc]
Hello guys, I'm a newbie of front end and need to debug (set breakpoints , step into the code etc. ) quasar app in webstorm.
But I don't know how to do ,where to begin. 
Is there a manual guid, instruction or something else for this?  Any helps are welcome , thanks!

**Comment by ontwikkelfabriek** [comment]
@luxi78 

Webstorm / any-other-IDE has marginal support for Vue SFC debugging let alone Quasar. It can work , sometimes, mostly not.
Use Chrome/Firefox inspector, will always work and that's the place where you have to deal with css/html debugging stuff anyways..
